components/org.wso2.identity.outbound.adapter.websubhub/src/test/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterUtilTest.java

```diff
@@ -329,6 +358,110 @@ public void testMakeTopicMgtAPICall(String topic, String webSubHubBaseUrl, Strin
         }
     }
 
+    @DataProvider(name = "makeAsyncAPICallInvalidParameterDataProvider")
+    public Object[][] makeAsyncAPICallInvalidParameterDataProvider() {
+
+        return new Object[][]{
+                // topic, web sub hub baseUrl, orgId, orgName, eventUri, testProperty, expectedException, expectedError
+                {null, WEBSUB_HUB_BASE_URL, TEST_ORG_ID, TEST_ORG_NAME, TEST_EVENT, TEST_PROPERTY,
+                        WebSubAdapterClientException.class, ERROR_INVALID_EVENT_TOPIC},
+                {TEST_TOPIC, null, TEST_ORG_ID, TEST_ORG_NAME, TEST_EVENT, TEST_PROPERTY,
+                        WebSubAdapterClientException.class, ERROR_INVALID_WEB_SUB_HUB_BASE_URL},
+                {TEST_TOPIC, WEBSUB_HUB_BASE_URL, TEST_ORG_ID, null, TEST_EVENT, TEST_PROPERTY,
+                        WebSubAdapterClientException.class, ERROR_INVALID_EVENT_ORGANIZATION_NAME},
+                {TEST_TOPIC, WEBSUB_HUB_BASE_URL, TEST_ORG_ID, TEST_ORG_NAME, null, TEST_PROPERTY,
+                        WebSubAdapterClientException.class, ERROR_INVALID_EVENT_URI},
+                {TEST_TOPIC, WEBSUB_HUB_BASE_URL, TEST_ORG_ID, TEST_ORG_NAME, TEST_EVENT, null,
+                        WebSubAdapterClientException.class, ERROR_NULL_EVENT_PAYLOAD},
+        };
+    }
+
+    @Test(dataProvider = "makeAsyncAPICallInvalidParameterDataProvider")
+    public void testMakeAsyncAPICallHandleInvalidParameters(String topic, String webSubHubBaseUrl,
+                                                            int orgId, String orgName, String eventUri,
+                                                            String testProperty, Class<?> expectedException,
+                                                            WebSubHubAdapterConstants.ErrorMessages error) {
+
+        String ref = "https://localhost:9443/" + orgName + "/test-event";
+
+        EventPayload testEvenPayload = getEventPayload(orgId, orgName, testProperty, ref);
```

Comment:
Typo in the variable name

Reply:
REsolves via [a292b74](https://github.com/wso2-extensions/identity-outbound-adapters/pull/28/commits/a292b749f8e23b8305c403050718c828e746a604)